frontend/src/container/IngestionSettings/MultiIngestionSettings.tsx

```diff
@@ -684,50 +732,108 @@ function MultiIngestionSettings(): JSX.Element {
 																	<div className="signal-limit-edit-mode">
 																		<div className="daily-limit">
 																			<div className="heading">
-																				<div className="title"> Daily limit </div>
+																				<div className="title">
+																					Daily limit
+																					<div className="limit-enable-disable-toggle">
+																						<Form.Item name="enableDailyLimit">
+																							<Switch
+																								size="small"
+																								checked={activeSignal?.config?.day?.enabled}
```

Comment:
nit: can we extract `activeSignal?.config?.day?.enabled` to a variable e.g. `isDailyLimitEnabled`

Reply:
Will take this up in the next PRs as we revamp the design @ahmadshaheer.